$NetBSD: patch-include_memory_h,v 1.1 2011/08/29 00:36:05 dholland Exp $

- fix use of inline semantics for the benefit of compilers that
pretend to be gcc but aren't.

--- include/memory.h~	2007-02-28 04:17:43.000000000 +0000
+++ include/memory.h
@@ -49,7 +49,7 @@ extern void _DupString(char **x, const c
 /* forte (and maybe others) don't like double declarations, 
  * so we don't declare the inlines unless GNUC
  */
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__GNUC_STDC_INLINE__)
 extern inline void *
 MyMalloc(size_t size)
 {
